spec/features/attendances/delete_zoom_attendance_spec.rb

```diff
@@ -0,0 +1,40 @@
+require 'rails_helper'
+
+RSpec.describe 'delete zoom attendance' do
+  include ApplicationHelper
+
+  before :each do
+    @user = mock_login
+    @module = create(:turing_module)
+    @test_attendance = create(:zoom_attendance, turing_module: @module)
```

Comment:
Using this factory will create an attendance with no associated student_attendances which is not realistic for production. Instead, use the trait `with_student_attendances` to create those associated records. After you do that, you should see that your test fails because you can't delete the attendance without deleting it's associated student attendances.

Reply:
Updated the :zoom_attendance with :attendance_with_student_attendances 
but all test still passed locally even with the change